### PR TITLE
Update german translation to v12.5.0

### PR DESCRIPTION
**Pull request by LinkDragmire**
Added new translations for v12.5 and done some minor fixes

### CODE REVIEW

File: lang/de.json [hunk 1]
```diff
@@ -439,8 +440,18 @@
             "TotalSkill": "Gesamt Fertigkeiten",
             "TotalProfSkill": "Gesamt Berufsfertigkeiten",
             "SpecialSkillModifier": "besondere Fähigkeits-Modifikatoren",
+			"Lebenspfad": {
```
Comment (Stexinator) [inline lang/de.json:443]: I think this will not work

File: lang/de.json [hunk 2]
```diff
@@ -189,7 +190,7 @@
             "Skill": "Durchführen eines Fähigkeitschecks",
             "profession.skill": "Durchführen einer Berufsfertigkeit"
         },
-	"DamageType": {
+	    "DamageType": {
```
Comment (Stexinator) [inline lang/de.json:193]: White spaces

File: lang/de.json [hunk 3]
```diff
@@ -181,6 +181,7 @@
             "attackWith": "Attacke durchführen mit",
             "chooseAmmunition": "Munition wählen",
             "NoAmmunition": "Keine Munition verfügbar",
+			"NoThrowable": "Keine Wurfwaffe verfügbar",
```
Comment (Stexinator) [inline lang/de.json:184]: White spaces

File: lang/de.json [hunk 4]
```diff
@@ -1225,7 +1236,61 @@
...
+            },
+            "chat": {
+                "requestTitle": "bittet um Reparatur"
+            },
+            "alerts": {
+                "notEnoughComponents": "Nicht genügend Komponenten, um diesen Gegenstand zu reparieren",
+                "itemIsAlreadyRepaired": "Dieser Gegenstand ist bereits vollständig repariert.",
+                "noDiagram": "Für diesen Gegenstand wurde kein zugehöriges Diagramm gefunden",
+                "invalidDiagram": "Der Typ dieses Diagramms ist für diesen Gegenstand nicht gültig"
+			}
```
Comment (Stexinator) [inline lang/de.json:1293]: Formatting

**Review by Stexinator — CHANGES_REQUESTED** [review]

File: lang/de.json (hunk 1 above)
Comment (LinkDragmire) [reply lang/de.json:443]: my bad, one replacement too much

**Review by Stexinator — APPROVED** [review]
Can you squash it? Then I merge it

**Comment by LinkDragmire** [comment]
Will be able to do so, this night

**Comment by LinkDragmire** [comment]
I'm not able to squash

**Comment by Stexinator** [comment]
You needed to squash it before you pull dev into it

**Pull request merged**